### PR TITLE
✨ Include ffmpeg in app, only send wav to backend

### LINKED ISSUE #47: bundle ffmpeg
currently we depend on the ffmpeg bundled with the system. sandboxed application, like the mac app, cannot access this. also we should provide one if we depend on it for systems which do not have one installed

### PR DESCRIPTION
**Pull request by pajowu**
This removed the ffmpeg dependency in the server component and fixes #47

### CODE REVIEW

**Review by anuejn — CHANGES_REQUESTED** [review]
Other than this tiny nit, looks good

**Comment by pajowu** [comment]
With this PR, import works. But as i noticed in #60 , export does not, as it depends on ffprobe. Should I move this back to draft and bundle ffprobe next week, or merge this for now and create a new PR for ffprobe?

**Comment by anuejn** [comment]
Hm... why / how does this depend on ffprobe exactly?

**Comment by pajowu** [comment]
ffmpeg-fluents `mergeToFile` uses ffprobe to figure out if the source files contain audio and/or video and set the `complexFilter` accordingly

File: app/src/main_process/index.ts [hunk 1]
```diff
@@ -55,6 +55,13 @@ const createWindow = (): void => {
     {
       label: 'Audapolis',
       submenu: [
+        {
```
Comment (anuejn) [inline app/src/main_process/index.ts:58]: can you split the new window stuff into a seperate commit?

File: app/src/exporters/ffmpeg.ts [hunk 2]
```diff
@@ -74,3 +81,26 @@ export async function exportContent(
   await combineParts(files, output_path, tempdir);
   fs.rmdirSync(tempdir, { recursive: true });
 }
+
+export async function convertToWav(input_path: string): Promise<Buffer> {
```
Comment (anuejn) [inline app/src/exporters/ffmpeg.ts:85]: in the future, we probably want to interleave this with the transcription but for now it seems fine

**Review by anuejn — APPROVED** [review]

File: app/src/main_process/index.ts (hunk 1 above)
Comment (pajowu) [reply app/src/main_process/index.ts:58]: done

**Pull request merged**